DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/SwiftUI/DisplayList+Reflection.swift

```diff
@@ -152,8 +152,12 @@ extension DisplayList.Content.Value: Reflection {
         case let (.enum("image"), image):
             self = try .image(reflector.reflect(image))
 
-        case (.enum("drawing"), _):
-            self = .unknown
+        case let (.enum("drawing"), (contents, origin, _) as (NSObject, CGPoint, Any)):
```

Comment:
See [this comment](https://github.com/DataDog/dd-sdk-ios/pull/2386#pullrequestreview-3000710074) on #2386